migration-from-froala.md

```diff
@@ -0,0 +1,485 @@
+  <!-- Emoticons plugin -->
+  <script type='text/javascript' src='/path/to/froala-editor/base/directory/js/plugins/emoticons.min.js'></script>
+  <link href='../css/plugins/emoticons.min.css' rel='stylesheet' type='text/css' />
+  <!-- Font Family plugin -->
+  <script type='text/javascript' src='/path/to/froala-editor/base/directory/js/plugins/font_family.min.js'></script>
+  <!-- Font Size plugin -->
+  <script type='text/javascript' src='/path/to/froala-editor/base/directory/js/plugins/font_size.min.js'></script>
+  <!-- Help plugin -->
+  <script type='text/javascript' src='/path/to/froala-editor/base/directory/js/plugins/help.min.js'></script>
+  <link href='/path/to/froala-editor/base/directory/css/plugins/help.min.css ' rel='stylesheet' type='text/css' />
+</head>
+```
+
+#### TinyMCE 5 - Enabling Plugins
+
+```html
+<script>
+  tinymce.init({
+    selector: '#tiny_editor',
+    plugins: 'emoticons wordcount help code lists'
+  });
+</script>
+```
+
+### Additional Information on Plugins
+* For information on enabling plugins for TinyMCE, see: [Integration and setup options: plugins]({{site.baseurl}}/configure/integration-and-setup/#plugins).
+* For a list of TinyMCE plugins, see: [Add plugins to TinyMCE]({{site.baseurl}}/plugins/).
+
+## Migrating a Custom Toolbar Layout
+
+To migrate a Custom Toolbar Layout from Froala to TinyMCE:
+* Change the `toolbarButtons` option to `toolbar`.
+* Update the toolbar item listing.
+* Update the `plugin` option as required.
+
+### Change the `toolbarButtons` option to `toolbar`
+Rename the Froala option `toolbarButtons` to `toolbar`.
+### Update the toolbar item listing
+Froala and TinyMCE use different formats for listing toolbar items.
+
+Froala accepts the list of toolbar buttons as a two-dimensional array of strings, with each array defining a group of toolbar items. For example:
+```js
+toolbarButtons: [['undo', 'redo'], ['paragraphFormat'], ['bold', 'italic'],
+['alignLeft', 'alignCenter', 'alignRight', 'alignJustify'],
+['formatOL', 'formatUL'], ['indent', 'outdent']]
+```
+TinyMCE accepts a space-delimited string with horizontal bars (` | `) for grouping items.
+For example:
+```js
+toolbar: 'undo redo | formatselect | bold italic
+| alignleft aligncenter alignright alignjustify
+| numlist bullist | outdent indent'
+```
+
+Replace the names of toolbar items with the TinyMCE names. For example:
+
+Toolbar Button | Froala | TinyMCE
+---|---|---
+Format/Style Selector | `paragraphFormat` | `formatselect`
+Ordered list | `formatOl` | `numlist`
+Unordered list | `formatUL` | `bullist`
+
+### Update the `plugin` option as required
+
+The list of plugins may need updating, as indicated in [Updating the list of Plugins]({{site.baseurl}}/migration-from-froala/#UpdatingthelistofPluginswhenmigratingfromFroala). For example, the TinyMCE toolbar items; `numlist` and `bullist`, require the `lists` plugin.
```

Comment:
Another one that could just be an anchor based link since it's in the same page. Also should this have a captial `U`, as I thought all id's were lowercase in the Jekyll output.